static/app/views/alerts/create.tsx

```diff
@@ -72,14 +75,15 @@ class Create extends Component<Props, State> {
   componentDidMount() {
     const {organization, project} = this.props;
 
-    trackAdvancedAnalyticsEvent('new_alert_rule.viewed', {
+    this.props.setRouteAnalyticsParams({
       organization,
```

Comment:
@roggenkemper i don't think we need `organization`

Reply:
you're right, ill remove!